actors/miner/src/monies.rs

```diff
@@ -267,10 +271,40 @@ pub fn initial_pledge_for_power(
     let lock_target_denom = LOCK_TARGET_FACTOR_DENOM;
     let pledge_share_num = qa_power;
     let network_qa_power = network_qa_power_estimate.estimate();
-    let pledge_share_denom = cmp::max(cmp::max(&network_qa_power, baseline_power), qa_power);
+
+    // Once FIP-0081 has fully activated, additional pledge will be 70% baseline
+    // pledge + 30% simple pledge.
+    const FIP_0081_ACTIVATION_PERMILLE: i64 = 300;
+    // Gamma is the share of pledge coming from the baseline formulation, with 1-gamma coming from simple pledge.
+    // Gamma ranges from 1.0 before the ramp to 0.7 at the end.
```

Comment:
This inconsistent definition of gamma is confusing.  If you want to refer to the FIP definition of Gamma alongside this code I think you should be calling your variable gamma_num since its the numerator.

Reply:
Alternatively you could change this to say gamma ranges from 700 to 1000 and then explain that the whole thing is divided by 1000 at the end so the share of simple vs baseline pledge ends up at 30% 70%.

Reply:
I changed the comment to highlight that gamma is per mille.